lib/grpfp.gi

```diff
@@ -3720,7 +3720,8 @@ local fgens,grels,max,gens,t,Attempt;
   t:=MostFrequentGeneratorFpGroup(G);
   gens:=Concatenation([t,
     #pseudorandom element - try if it works
-    Product([1..Random([2,3])],i->Random(gens)^Random([1,-1]))],
+    PseudoRandom(G:radius:=Random(2,3))],
+    #Product([1..Random([2,3])],i->Random(gens)^Random([1,-1]))],
```

Comment:
This change is introduced by the commit adding `ProcessEpimorphismToNewFpGroup` -- an accident?

Reply:
I spotted this while going over code that uses homomorphisms with fp groups. It seemed to small a change to deserve its own commit. (The `PseudoRandom` is cleaner than the `Random(gens)^Random([-1,1]) hack.)

Reply:
OK, so then, remove the old code, and mention it in the commit message?